templates/tips/edit_tip.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %} {%load static%} {%block content%}
+<div class="editfrm">
+	<p class="edittip"><h1>Edit tip</h1></p>
+	<form method="POST">
+		{% csrf_token %}
+		{{ form.as_p }}
+		<input type="submit" value="Post">
```

Comment:
Do you show the current version of the tip

Reply:
It does!